package.json

```
@@ -1,5 +1,5 @@
 {
-    "version": "7.7.1-penbase-beta-2",
+    "version": "7.8.1-penbase-beta-1",
```

```suggestion
    "version": "7.8.1-penbase",
```

Même commentaire sur le suffixe beta-1. A toi de voir si tu en as besoin et uniformiser (Sans oublier le package-lock).